invokeai/app/invocations/prompt.py

```diff
@@ -55,3 +56,35 @@ def invoke(self, context: InvocationContext) -> PromptCollectionOutput:
             prompts = generator.generate(self.prompt, num_images=self.max_prompts)
 
         return PromptCollectionOutput(prompt_collection=prompts, count=len(prompts))
+    
+
+class PromptsFromFileInvocation(BaseInvocation):
+    '''Loads prompts from a text file'''
+    # fmt: off
+    type: Literal['prompt_from_file'] = 'prompt_from_file'
+
+    # Inputs
+    filename: str = Field(default=None, description="Filename of prompt text file")
+    pre_prompt: str = Field(default=None, description="Add to start of prompt")
+    post_prompt: str = Field(default=None, description="Add to end of prompt")
+    start_line: int = Field(default=1, ge=1, description="Line in the file start start from")
+    max_prompts: int = Field(default=0, ge=0, description="Max lines to read from file (0=all)")
```

Comment:
- might be a good idea to make the default like 100 or 1000, instead of 0, just as a sanity check kinda thing